### PR TITLE
Tidy configs and files

### PR DESCRIPTION
**Pull request by gmrukwa**
Some cleanups: reorganization od additional files in repo.

Special case for test also prepared (smaller).

**Note**: *This branch has been started from WPF client.*

### CODE REVIEW

File: src/Spectre.Algorithms.Tests/Methods/SegmentationTests.cs [hunk 1]
```diff
@@ -68,7 +68,7 @@ public void DivikSimple()
         public void DivikBigData()
         {
             // path to directory with test project
-            var path = TestContext.CurrentContext.TestDirectory + "\\..\\..\\..\\single.txt";
+            var path = TestContext.CurrentContext.TestDirectory + "\\..\\..\\..\\..\\..\\test_files\\hnc1_tumor.txt";
```
Comment (Sebaxp95) [inline src/Spectre.Algorithms.Tests/Methods/SegmentationTests.cs]: Maybe \\..\\..\\..\\..\\..\\ put is some string e.g. main directory?

Comment (gmrukwa) [inline src/Spectre.Algorithms.Tests/Methods/SegmentationTests.cs]: Reasonable remark. Solved :)

**Comment by FreexD** [comment]
LGTM. Good job :1st_place_medal:

**Comment by gmrukwa** [comment]
[Spring cleanup](https://trello.com/c/mOjn9wKQ/94-spring-cleanup)

**Pull request merged**